### ISSUE TITLE
Cannot get capacity data on HP Ultrium LTO-6

### ISSUE THREAD

**Issue by bigfatbrowncat** [LinearTapeFileSystem/ltfs#450]
**Describe the bug**
I can not format a cartridge. The device is HP Ultrium LTO-6.
The project is built with `configure --enable-buggy-ifs`.
Whatever I try to run, I get
```
LTFS30232I Cannot get remaining capacity: failed to parse the log page.
LTFS12030E Cannot get capacity data: backend call failed (-21700).
LTFS11999E Cannot load the medium: failed to get capacity data (-21700).
LTFS11331E Failed to load the cartridge (ltfs_load_tape).
```

**To Reproduce**
Build and install the project on Debian 12.

**Expected behavior**
The cartridge should be formatted

**Screenshots**
The log:
```
$ sudo mkltfs -d HU1328WFYB -t
LTFS15000I Starting mkltfs, LTFS version 2.4.5.1 (Prelim), log level 3.
LTFS15041I Launched by "mkltfs -d HU1328WFYB -t".
LTFS15042I This binary is built for Linux (x86_64).
LTFS15043I GCC version is 12.2.0.
LTFS17087I Kernel version: Linux version 6.1.0-20-amd64 (debian-kernel@lists.debian.org) (gcc-12 (Debian 12.2.0-14) 12.2.0, GNU ld (GNU Binutils for Debian) 2.40) #1 SMP PREEMPT_DYNAMIC Debian 6.1.85-1 (2024-04-11) i386.
LTFS17089I Distribution: PRETTY_NAME="Debian GNU/Linux 12 (bookworm)".
LTFS15025D Validating command line options.
LTFS15037D Command line options are valid.
LTFS15003I Formatting device 'HU1328WFYB'.
LTFS15004I LTFS volume blocksize: 524288.
LTFS15005I Index partition placement policy: None.

LTFS11337I Update index-dirty flag (1) - NO_BARCODE (0x0x564dcce69ea0).
LTFS15006D Opening the device.
LTFS17085I Plugin: Loading "sg" tape backend.
LTFS30209I Opening a device through sg-ibmtape driver (HU1328WFYB).
LTFS30288I Opening a tape device for drive serial HU1328WFYB.
LTFS30250I Opened the SCSI tape device 0.0.0.0 (/dev/sg3).
LTFS30207I Vendor ID is HP      .
LTFS30208I Product ID is Ultrium 6-SCSI  .
LTFS30214I Firmware revision is J5IW.
LTFS30215I Drive serial is HU1328WFYB.
LTFS30290I Changer /dev/sg3 isn't reserved from any nodes.
LTFS30285I The reserved buffer size of HU1328WFYB is 1048576.
LTFS30294I Setting up timeout values from RSOC.
LTFS12023D Reserving device.
LTFS30392D Backend reserve (PRO) HU1328WFYB.
LTFS12028D Unlocking medium.
LTFS30392D Backend allow medium removal HU1328WFYB.
LTFS30392D Backend read block limits HU1328WFYB.
LTFS17160I Maximum device block size is 1048576.
LTFS11330I Loading cartridge.
LTFS30392D Backend load HU1328WFYB.
LTFS30398D Backend readpos: (0, 0) FM = 0 HU1328WFYB.
LTFS12026D Locking medium in the drive.
LTFS30392D Backend prevent medium removal HU1328WFYB.
LTFS30398D Backend readpos: (0, 0) FM = 0 HU1328WFYB.
LTFS30232I Cannot get remaining capacity: failed to parse the log page.
LTFS12030E Cannot get capacity data: backend call failed (-21700).
LTFS11999E Cannot load the medium: failed to get capacity data (-21700).
LTFS11331E Failed to load the cartridge (ltfs_load_tape).
LTFS30232I Cannot get remaining capacity: failed to parse the log page.
LTFS12030E Cannot get capacity data: backend call failed (-21700).
LTFS11999E Cannot load the medium: failed to get capacity data (-21700).
LTFS15044E Cannot set up tape device.
LTFS12028D Unlocking medium.
LTFS30392D Backend allow medium removal HU1328WFYB.
LTFS12025D Releasing device.
LTFS30392D Backend release (PRO) HU1328WFYB.
LTFS30393D Backend LBP Enable: 0 .
LTFS30393D Backend LBP Method: 1 .
LTFS30201D CDB check condition: sense = 052600, Invalid Field in Parameter List.
LTFS30205I MODESELECT (0x55) returns -20500.
LTFS30263I MODESELECT returns Invalid Field in Parameter List (-20500) /dev/sg3.
LTFS15023I Formatting failed.
```

**Environment:**
 - OS: Debian 12 (Bookworm)
 - HP Ultrium LTO-6

**Comment by piste-jp-ibm** [comment]
Please search the issues by `LTFS12030E`. I found other 4 issues and my comment in #444 might be an answer.

---
I think this is already reported in #241 and fixed on the master branch with #248.

The fix is only available on the master.

You can see also #418.
---

**Comment by piste-jp** [comment]
Close this because there is no activity long time.

Please feel free to reopen if something new is found.
